crane/views/v2.py

```diff
@@ -99,6 +100,9 @@ def name_redirect(relative_path):
                 path_component = os.path.join(manifest, '2', identifier)
             else:
                 path_component = os.path.join(manifest, '1', identifier)
+        # this is needed for V3Repo which do not have schema2 manifests
```

Comment:
It looks like the V3Repo does have schema 2 manifests. Can you clarify?

``V3Repo = namedtuple('V3Repo', ['url', 'url_path', 'schema2_data', 'protected'])``